apps/andi/assets/package-lock.json

```diff
@@ -1,5478 +1,15 @@
 {
-  "name": "assets",
-  "lockfileVersion": 2,
   "requires": true,
-  "packages": {
```

Comment:
Did you do this on purpose? Like a prune that had to occur?

Reply:
no, just what happened when I built. I typically try to leave it out, but I think I had to, as I added a couple imports